docs/sources/static/release-notes.md

```diff
@@ -15,16 +15,18 @@ The release notes provide information about deprecations and breaking changes in
 
 For a complete list of changes to Grafana Agent, with links to pull requests and related issues when available, refer to the [Changelog](https://github.com/grafana/agent/blob/main/CHANGELOG.md).
 
-{{% admonition type="note" %}}
-These release notes are specific to Grafana Agent static mode. 
-Other release notes for the different Grafana Agent variants are contained on separate pages:
-
-* [Static mode Kubernetes operator release notes][release-notes-operator]
-* [Flow mode release notes][release-notes-flow]
-
-[release-notes-operator]: {{< relref "../operator/release-notes.md" >}}
-[release-notes-flow]: {{< relref "../flow/release-notes.md" >}}
-{{% /admonition %}}
+> **Note:** These release notes are specific to Grafana Agent static mode. 
+> Other release notes for the different Grafana Agent variants are contained on separate pages:
+>
+> * [Static mode Kubernetes operator release notes][release-notes-operator]
+> * [Flow mode release notes][release-notes-flow]
+
+{{% docs/reference %}}
+[release-notes-operator]: "/docs/agent/ -> /docs/agent/<AGENT VERSION>/operator/release-notes"
+[release-notes-operator]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/monitor-infrastructure/integrations/agent/operator/release-notes"
+[release-notes-flow]: "/docs/agent/ -> /docs/agent/<AGENT VERSION>/flow/release-notes"
+[release-notes-flow]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/release-notes"
```

Comment:
So I believe the intent is that we will unmount the Flow documentation from Grafana Cloud which means this page won't exist. Do we want to link back to the Agent docs instead?

Alternatively, we can leave Flow release notes in Grafana Cloud and exclude everything else.

Reply:
Can we direct back out of Cloud into the Agent docs for this link? Maybe not a good idea since it'll push the reader out of the Cloud doc tree if they click the link... but it's a short term solution either way.